### PR TITLE
RFC14 Autocomplete Suggestions (Front-End) - 3.10 Release, Minor Fixes

### PR DESCRIPTION
**Pull request by hadrian-git**
Minor fixes following previous PR review, #6662

### CODE REVIEW

File: web/src/search/input/QueryInput.tsx [hunk 1]
```diff
@@ -396,7 +398,7 @@ export class QueryInput extends React.Component<Props, State> {
                                         {this.state.loadingSuggestions && (
                                             <li className="suggestion suggestion--selected">
                                                 <LoadingSpinner className="icon-inline" />
-                                                <div className="suggestion__description">fetching suggestions...</div>
+                                                <div className="suggestion__description">LOADING</div>
```
Comment (attfarhan) [inline web/src/search/input/QueryInput.tsx]:
```suggestion
                                                <div className="suggestion__description">Loading</div>
```

Comment (hadrian-git) [reply web/src/search/input/QueryInput.tsx]: I set it capitalized based on this [comment](https://github.com/sourcegraph/sourcegraph/pull/6662#discussion_r347907186), maybe I understood it wrong?

Comment (attfarhan) [reply web/src/search/input/QueryInput.tsx]: I think he means capitalized as in, the first letter should be capitalized, not the whole thing. See other places we do loading indicators: https://sourcegraph.com/search?q=r:sourcegraph/sourcegraph%24+Loading...+case:yes+f:%5C.tsx&patternType=literal.

You actually might want to add an ellipses too: 

```suggestion
                                                <div className="suggestion__description">Loading...</div>
```

Comment (felixfbecker) [inline web/src/search/input/QueryInput.tsx]: Sorry - when I said capitalized, I meant "Loading" 😅 
```suggestion
                                                <div className="suggestion__description">Loading</div>
```

Comment (hadrian-git) [reply web/src/search/input/QueryInput.tsx]: Ah, I understood uppercase. Fixed :+1:

**Pull request merged**